### ISSUE TITLE
Interactive plotting is not working during debugging

### ISSUE THREAD

**Issue by spyder-issues-migrator** [spyder-ide/spyder#1848]
_From [jonasnis...@gmail.com](https://code.google.com/u/112746919483377611443/) on 2014-06-11T15:04:53Z_

This issue (improvement suggestion) was discussed with Almar Klein for IEP. The issue (and Almar's implementation) can be seen at: https://bitbucket.org/iep-project/iep/issue/297/interactive-plotting-is-not-working-during As I also use Spyder, it could be awesome if a similar functionality could be implemented for Spyder as well :-)

_Original issue: http://code.google.com/p/spyderlib/issues/detail?id=1848_

**Comment by spyder-issues-migrator** [comment]
_From [dive...@gmail.com](https://code.google.com/u/101061052488374976516/) on 2014-06-15T11:07:33Z_

I support this suggestion. I implemented the above small change suggested by Almar Klein into IEP and I enjoyed for the first time the plots being updated during Python debugging, as I am used to see in IDL or MATLAB. It would be great if this functionality could be included in Spyder too.
